be/src/pipeline/task_scheduler.cpp

```diff
@@ -278,8 +278,8 @@ void TaskScheduler::_do_work(size_t index) {
 
         task->set_previous_core_id(index);
         if (!status.ok()) {
```

Comment:
I think warning should be used here, because `status` may return some internal errors

Reply:
how do you think if we print a WARNING message of PipelineTask failure, and print verbose information in INFO?